### ISSUE TITLE
[8.0] base_report_to_printer: odoo server does not start if cups is not available

### ISSUE THREAD

**Issue by eLBati** [OCA/report-print-send#26]
Related to https://github.com/OCA/l10n-italy/issues/33#issuecomment-72636010

Try removing `cups` python module and start odoo server.
Get

```
  File "/home/elbati/workspace/odoo/buildout_cfg-odoo-8/parts/report-print-send/base_report_to_printer/__init__.py", line 24, in <module>
    from . import printing
  File "/home/elbati/workspace/odoo/buildout_cfg-odoo-8/parts/report-print-send/base_report_to_printer/printing.py", line 27, in <module>
    import cups
ImportError: No module named cups
```

Also, I tried removing `ldap` python module but odoo module `auth_ldap` (that uses `external_dependencies` too) does not raise exception while starting odoo.
It does not even load `users_ldap.py` at start time. So, when trying to install `auth_ldap`, the system correctly says
`Unable to install module "auth_ldap" because an external dependency is not met: No module named ldap`

So, there are differences between `auth_ldap` and `base_report_to_printer` about how they are loaded. But I can't find such differences at the moment

**Comment by pedrobaeza** [comment]
You can try to enclose the import in a try, and make sure that the calls to this lib are inside code that it's not invoked in the server initialisation.

**Comment by eLBati** [comment]
Some extra info: https://www.odoo.com/groups/community-59/community-11511562

**Comment by RoelAdriaans** [comment]
Just created a pull request to fix this, number #43 
See https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md , External dependencies
